documentation/modules/auxiliary/scanner/http/f5_mgmt_scanner.md

```diff
@@ -0,0 +1,32 @@
+## Vulnerable Application
+
+This module scans for web management interfaces of the following F5 Networks devices:
```

Comment:
The English here looks a bit off, would it not be better to say "This module attempts to identify the web management interfaces of the following F5 Networks devices:"?

Reply:
also updated the language in the module itself since this was a copy pasta from there